bot/Dockerfile

```
@@ -5,7 +5,7 @@ ADD bot /src/bot
 
 RUN /src/bot/ci/bootstrap.sh
 
-RUN cd /src/tools && python setup.py install
-RUN cd /src/bot && python setup.py install
+RUN cd /src/tools && pip install --disable-pip-version-check --no-cache-dir --quiet . 
```

Please remove the trailing whitespace

```suggestion
RUN cd /src/tools && pip install --disable-pip-version-check --no-cache-dir --quiet .
```